install.yml

```
@@ -336,8 +336,37 @@ onAfterClone:
       res = jelastic.dev.scripting.eval("appstore", session, "InstallApp", data);
       return {"result": 0, "out": res}
 
+onBeforeDelete:
+  deleteEnvLink
+
 success: /text/success.md
 
+# -- Actions --
+actions:
+  deleteEnvLink:
+    - script: |-
+        const envName = "${env.envName}";
+        jCustomerEnv = jelastic.env.control.GetNodeGroups(envName, session).object.filter(function (object) {
+                                return object.name == "cp";
+                              }).pop().envLink;
+        if (jCustomerEnv != null) {
+          envsLinked = jelastic.env.control.GetNodeGroups(jCustomerEnv, session).object.filter(function (object) {
+                                return object.name == "cp";
+                              }).pop().envLink;
+          if (envsLinked.indexOf(",") != -1) {
+            envsLinkedArr = envsLinked.split(",");
+            removedEnv = envsLinkedArr.splice(envsLinkedArr.indexOf[envName], 1);
+            newEnvLink = String(envsLinkedArr);
+            resp = jelastic.env.control.ApplyNodeGroupData(jCustomerEnv, session, nodeGroup='cp', data={'envLink': newEnvLink});
+            return {"result": 0, "out": "Jahia env " + removedEnv + " removed from envLink of " + jCustomerEnv};
+          } else {
+              resp = jelastic.env.control.ApplyNodeGroupData(jCustomerEnv, session, nodeGroup='cp', data={'envLink': null});
+              return {"result": 0, "out": "Jahia env " + envName + " removed from envLink of " + jCustomerEnv};
+          }
```

We can avoid duplicating code like this (also fixes indenting issue):
```suggestion
            newEnvLink = String(envsLinkedArr);
          } else {
            newEnvLink = null;
          }
          resp = jelastic.env.control.ApplyNodeGroupData(jCustomerEnv, session, nodeGroup='cp', data={'envLink': newEnvLink});
          return {"result": 0, "out": "Jahia env " + envName + " removed from envLink of " + jCustomerEnv};
```